core/pkg/filestream/loop_process.go

```diff
@@ -69,6 +69,10 @@ func (fs *FileStream) loopProcess(inChan <-chan protoreflect.ProtoMessage) {
 }
 
 func (fs *FileStream) streamHistory(msg *service.HistoryRecord) {
+	// msg.Item = append(msg.Item, &service.HistoryItem{
+	// 	Key:       "_client_id",
```

Comment:
looks like client id is only relevant for the stream proxy the regular async doesn't need this!